### ISSUE TITLE
[Feature Request] Add setting to not fold empty whitespace between sections.

### ISSUE THREAD

**Issue by mm-gmbd** [melke/markdown-folder#3]
Currently, whitespace between sections is folded. For example, the unfolded text

```
#Header1
Line 1 Text
Line 2 Text



#Header 2
Line 1 Text
Line 2 Text
```

Gets folded into

```
#Header1 --
#Header2 --
```

A settings option to disable folding of empty whitepsace would be nice (for some visual separation). With this option, the previous example would be folded as 

```
#Header1 --



#Header2 --
```

**Comment by melke** [comment]
Fixed in 0.1.0 (now public on atom.io/packages)

**Comment by mm-gmbd** [comment]
Thanks, @melke!
